src/recon.rs

```diff
@@ -263,21 +265,25 @@ impl_MergeInt!(u32, u16);
 impl_MergeInt!(u64, u32);
 impl_MergeInt!(u128, u64);
 
-#[inline]
-fn get_skip_ctx(
-    t_dim: &TxfmInfo,
+#[inline(always)]
+fn get_skip_ctx<const txsize_discr: usize>(
```

Comment:
```suggestion
fn get_skip_ctx<const TX: usize>(
```

I think this is the standard naming for a `TxfmSize`.

Reply:
I've pushed a commit to do this renaming.